Formula/awsume.rb

```diff
@@ -9,6 +9,11 @@ class Awsume < Formula
   revision 1
   head "https://github.com/trek10inc/awsume.git"
 
+  livecheck do
+    url :head
+    regex(/v?(\d+(?:\.\d+)+)$/i)
```

Comment:
Sorry about that @SeekingMeaning, I'd missed the initial `^` in the regex as pointed out by Sam. I'll be more careful next time 👍🏼
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```